plugins/chat/lib/chat/guardian_extensions.rb

```diff
@@ -13,7 +13,7 @@ def can_moderate_chat?(chatable)
 
     def can_chat?
       return false if anonymous?
-      @user.staff? || @user.in_any_groups?(Chat.allowed_group_ids)
+      is_staff? || @user.bot? || @user.in_any_groups?(Chat.allowed_group_ids)
```

Comment:
the rename from `@user.staff?` to `is_staff?` is not related to the change, just felt more consistent